### DISCUSSION TITLE
Clickable area of previous month extends to full root div inside form labels

### DISCUSSION THREAD

**Discussion by rcoopr** [gpbl/react-day-picker#1500] [Support]
### Bug description

The hover and clickable area for the previous month button (and in dropdown mode, the month selector) extends to the entire calendar canvas, i.e. `<div class="rdp">` when used inside a label element

### To reproduce

https://codesandbox.io/s/react-day-picker-8-forked-8nbrzg

#### Steps

1. Hover near the bottom of the calendar
2. Observe the month back button highlight and button clickbox is active throughout entire background

**Comment by gpbl** [comment]
Thanks for letting us know! This is probably because the `button` HTML elements used inside the calendar. Why should you wrap them with `label`?
